src/lsqecc/simulation/lazy_tensor_op.py

```diff
@@ -38,7 +39,7 @@ class LazyTensorOp(Generic[T]):
     operands. Has methods to apply operators so that the tensor structure is preserved."""
 
     def __init__(self, ops: List[T]):
-        self.ops = ops
+        self.ops = [op for op in ops]  # TODO rename to operands
```

Comment:
```suggestion
        self.ops = ops[:]  # TODO rename to operands
```
I think something like this also works as well.